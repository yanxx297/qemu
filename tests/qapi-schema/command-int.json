# FIXME: we should reject collisions between commands and types
{ 'command': 'int', 'data': { 'character': 'str' },
  'returns': { 'value': 'int' } }
