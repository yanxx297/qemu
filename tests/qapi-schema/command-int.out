[OrderedDict([('command', 'int'), ('data', OrderedDict([('character', 'str')])), ('returns', OrderedDict([('value', 'int')]))])]
[]
[]
